wallet/src/repository/wallet/tests/mod.rs

```diff
@@ -372,16 +377,18 @@ fn test_create_transaction_components_without_a_change_address() {
     let value = 1;
     let fee = 0;
     let time_lock = 0;
-    let vtt = wallet
-        .create_vt_transaction_components(&mut state, value, fee, Some((pkh, time_lock)))
+    let vto = ValueTransferOutput {
+        pkh,
+        value,
+        time_lock,
+    };
+
+    let (inputs, outputs) = wallet
+        .create_vt_transaction_components(&mut state, vec![vto], fee)
         .unwrap();
 
-    assert_eq!(1, vtt.value);
-    assert_eq!(0, vtt.change);
-    assert_eq!(vec![out_pointer], vtt.used_utxos);
-    assert_eq!(1, vtt.sign_keys.len());
-    assert_eq!(1, vtt.inputs.len());
-    assert_eq!(1, vtt.outputs.len());
+    assert_eq!(1, inputs.len());
+    assert_eq!(1, outputs.len());
```

Comment:
We can test the output value, but I think we cannot test the fee

```suggestion
    assert_eq!(1, outputs.len());
    assert_eq!(value, outputs_sum(outputs));
```

Reply:
In the next PR of @girazoki, the logic fee changes and there will be so many tests checking fee logic, so maybe it is not really need it now